pkg/console/console.go

```diff
@@ -35,12 +35,13 @@ type UIConfig struct {
 
 // FrontendConfig is the configuration for the Console frontend.
 type FrontendConfig struct {
-	Language string          `json:"language" name:"-"`
-	IS       webui.APIConfig `json:"is" name:"is"`
-	GS       webui.APIConfig `json:"gs" name:"gs"`
-	NS       webui.APIConfig `json:"ns" name:"ns"`
-	AS       webui.APIConfig `json:"as" name:"as"`
-	JS       webui.APIConfig `json:"js" name:"js"`
+	Language    string          `json:"language" name:"-"`
+	SupportLink string          `json:"support_link" name:"support-link" description:"The URL pointing to a support website, shown as a button in the footer"`
```

Comment:
Technically it can be a `mailto:` link as well, and we probably see people using that

Reply:
So it's still `SupportLink`, but not not URL but URI (which doesn't have to be a website)